src/modules/muorb/apps/uORBAppsProtobufChannel.cpp

```diff
@@ -110,19 +157,23 @@ bool uORB::AppsProtobufChannel::Test(MUORBTestType test_type)
 
 	switch (test_type) {
 	case ADVERTISE_TEST_TYPE:
-		rc = fc_sensor_advertise(muorb_test_topic_name);
+		rc = topic_advertised(muorb_test_topic_name);
+		PX4_INFO("succesfully did ADVERTISE_TEST_TYPE");
 		break;
 
 	case SUBSCRIBE_TEST_TYPE:
-		rc = fc_sensor_subscribe(muorb_test_topic_name);
+		rc = add_subscription(muorb_test_topic_name, 1);
+		PX4_INFO("succesfully did SUBSCRIBE_TEST_TYPE");
 		break;
 
 	case TOPIC_TEST_TYPE:
-		rc = fc_sensor_send_data(muorb_test_topic_name, test_data, MUORB_TEST_DATA_LEN);
```

Comment:
Leave this one as fc_sensor_send_data so that you don't have to worry about whether there is a subscriber or not. This is just testing low level muorb mechanism so no need to verify that there are subscribers.